.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+trim_trailing_whitespace = true
+
+[*.{c,h,cpp}]
+indent_style = space
```

Comment:
I need @opdenkamp to review, our usual style is tab indenting, but might not be for this project

Reply:
Fair enough. I just based that on cec-client.cpp, cecc-client.c, etc.

I see that max line length doesn't seem to be enforced, so I just guessed 120